pkg/taxonomy/bad_catalog.structs.schema.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Is this meant to be a real schema or just used for testing the tool?  If the latter, perhaps it should be in a test data directory?